src/config/hecks.rs

```diff
@@ -19,7 +19,7 @@ impl Hecks {
         // Create a file if it does not exist
         if !Path::new(HECK_FILE_PATH).exists() {
             warn!("guild_settings.toml does not exist, creating it...");
-            contents = r"hecks = {}".to_string();
+            contents = "hecks = {}\nsfw_hecks = []\nnsfw_hecks = []\nsfw_heck_ids = []\nnsfw_heck_ids = []".to_string();
```

Comment:
This should all be `[]` for complete sake. Yeah, this code works, but I like consistency :)
```suggestion
            contents = "hecks = []\nsfw_hecks = []\nnsfw_hecks = []\nsfw_heck_ids = []\nnsfw_heck_ids = []".to_string();
```

Reply:
I agree completely, I just did [] since it was what the code already expected. I was only fixing bugs rather than polish. I'm happy to fix it, now I have more experience with the codebase.